tools/eslint/index.js

```diff
@@ -24,12 +24,10 @@ module.exports = {
     },
   },
   overrides: [
-    // JavaScript for Node.js
+    // Jest Test files
     {
-      files: ['src/backend/**/*.js', 'tools/**/*.js', 'src/api/**/*.js'],
-      env: {
-        node: true,
-      },
+      files: ['test/**/*.js', '*.test.js', '*.test.ts', '*.test.tsx', '**/__mocks__/**/*.js'],
+      env: { jest: true, node: true },
```

Comment:
I suspect this wasn't resolved properly. You can get rid of the entire `overrides: [ ... ]` block since this PR deals with the last sub-project with overrides. Once this PR lands, it will also close #2965

Reply:
@cindyledev yes, thanks.